zebra-consensus/src/transaction.rs

```diff
@@ -298,6 +302,29 @@ where
         )
     }
 
+    /// Verifies if a V4 `transaction` is supported by `network_upgrade`.
+    fn verify_v4_transaction_network_upgrade(
+        transaction: &Transaction,
+        network_upgrade: NetworkUpgrade,
+    ) -> Result<(), TransactionError> {
+        match network_upgrade {
+            // Supports V4 transactions
```

Comment:
Can you add a similar comment to the V5 check?

Reply:
Done in 6969c3109